cardano-node/src/Cardano/Node/Tracing/API.hs

```diff
@@ -86,7 +86,9 @@ initTraceDispatcher nc p networkMagic nodeKernel p2pMode = do
     EKG.registerGcMetrics ekgStore
     ekgTrace <- ekgTracer (Left ekgStore)
 
-    stdoutTrace <- standardTracer
+    -- We don't shutdown the trace layer in the node so we can ignore the
+    -- shutdown action
+    (stdoutTrace, _shutdown) <- standardTracer
```

Comment:
Should we ignore it? Note that the thread that is running it, is left stranded - it will be killed by RTS, which is not ideal.  Would it be fine to use it in `finally` handler of the main thread? The only concern is to not shut it down to early while still some messages are generated.